eos-core-depends

```diff
@@ -112,6 +112,7 @@ nautilus
 bluez-obexd
 openprinting-ppds
 openssh-server
+openvpn
```

Comment:
I think you want `network-manager-openvpn`, no? (We may not have the package built on OBS)